app/helpers/mailer_helper.rb

```diff
@@ -182,7 +182,7 @@ def style_work_tag_metadata(tags)
 
   def commenter_pseud_or_name_link(comment)
     if comment.comment_owner.nil?
-      style_bold(comment.comment_owner_name) + style_role(t("roles.guest_with_parens"))
+      "#{style_bold(comment.comment_owner_name)} #{style_role(t('roles.guest_with_parens'))}"
```

Comment:
How did you actually check the email output? I couldn't find where the emails are explained in the wiki.

Reply:
I set up a comment [mailer preview](https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails) for [AO3-6017](https://otwarchive.atlassian.net/browse/AO3-6017), which I'm working on right now. It needs the fix in #4680 to work locally.

You might also be able to find the emails in the logs or use the MailCatcher gem according to https://github.com/otwcode/otwarchive/pull/4475#pullrequestreview-1329628736. I find it much easier and faster to use the mailer previews since it doesn't require me to actually leave comments, so I haven't tried those options.